pkg/cache/basic/cache.go

```diff
@@ -8,11 +8,16 @@ import (
 	"github.com/bacalhau-project/bacalhau/pkg/util/generic"
 )
 
+const (
+	DefaultTTLSeconds = int64(3600)
```

Comment:
Let's move this to the default config structures.

Reply:
We need this here in case somebody calls NewCache() without specify the option that sets TTL - so this is just to appease the linter (although it doesn't complain about 1000 a few lines above). I think it'd make sense if ttl was a required parameter though.